### ISSUE TITLE
Trusts summon command doesn't work

### ISSUE THREAD

**Issue by vekien** [Windower/Lua#1684]
https://github.com/Windower/Lua/blob/live/addons/Trusts/README.md

- I saved a set `//tru save levelling`
- I try summon: `//tru levelling`

Nothing happens.

**Comment by from20020516** [comment]
hi viion.

I can't reproduce that issue. I need more information.
- `resources/key_items.lua` are newest one?
- `trusts/data/settings.xml` <language\>  are same as your client language?

**Comment by vekien** [comment]
I will try again tonight and double check I'm not doing something stupid.

- My `resources/key_items.lua` I am assuming is up to date? I downloaded/installed windower about a week ago
- `trusts/data/settings.xml` - I did not change anything here so I will check, i'm on NA client.

Could this break if you have a non-trust in the party? When I tried to use this i had a sub-account in my party. Would it attempt to register them as a trust? Going to test that tonight.
